app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<h1 class="text-center mb-2 text-xl font-extrabold text-gray-900  md:text-2xl lg:text-3xl">
+  Shopping list 
+</h1>
+<div class="flex flex-col gap-y-1">
+  <div class="flex justify-between">
+    <div class="text-xl font-semibold text-gray-900">
+      <span>Amount of Food to buy: </span> <span class="italic"><%= @missing_foods.length %></span>
+    </div>
+    <div class="text-xl font-semibold text-gray-900">
+      <span>Recipe: </span> <span class="italic underline"><%= link_to @recipe.name , recipe_path(@recipe) %></span>
+    </div>
+  </div>
+  <div class="flex justify-between">
+    <div class="text-lg font-semibold text-gray-900">
+      <span>Total value of food needed: </span> <span class="italic">$<%= @missing_foods.reduce(0) { |sum, f| sum + (f.price.to_i * f.quantity(@recipe).to_i) } %></span>
+    </div>
+    <div class="text-xl font-semibold text-gray-900">
+      <span>Inventory: </span> <span class="italic underline"><%= link_to @inventory.name, inventory_path(@inventory) %></span>
+    </div>
+  </div>
+  <%# table %>
+  <div class="relative overflow-x-auto shadow-md sm:rounded-lg">
+    <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
```

Comment:
- [ ] I think it would be better to add the  shopping_list page to your menu so  that users can see the shopping_list easily and not only when users generate the shopping_list. In your case, the user can see the shopping_list page only if he /she generates it and after exiting the page they can't see it anymore. :wink: 
<img width="843" alt="Screen Shot 2023-02-11 at 16 22 55 (1)" src="https://user-images.githubusercontent.com/88894525/218262104-8254ba80-ee8d-4630-8ff6-7db204108aac.png">